caas/kubernetes/provider/operator.go

```diff
@@ -590,6 +730,7 @@ func operatorConfigMap(appName, name string, labels, annotations map[string]stri
 			Name:        name,
 			Labels:      labels,
 			Annotations: annotations,
+			Generation:  config.ConfigMapGeneration,
```

Comment:
what happens when there's a Generation mismatch?
do we get an error from k8s that causes us to need to try again?
I can't see where we do that

Reply:
Handled like any other error. K8s would return a Conflict error, because the generation doesn't match what was read.

Reply:
The caasoperatorprovisioner retries already.